Multisig/UI/Assets/Collectibles/CollectiblesSectionView.swift

```diff
@@ -13,18 +13,25 @@ struct CollectiblesSectionView: View {
 
     private let imageDimention: CGFloat = 28
     var body: some View {
-        VStack (alignment: .leading, spacing: 12) {
-            HStack {
-                TokenImage(width: imageDimention,
-                           height: imageDimention,
-                           url: section.imageURL,
-                           name: "ico-nft-placeholder")
-                Text(section.name).headline()
-            }
-
+        Section (header: header) {
             ForEach(section.collectibles) { collectible in
-                CollectibleCellView(viewModel: collectible)
+                NavigationLink(destination: CollectibleDetailsView(viewModel: collectible)) {
+                    CollectibleCellView(viewModel: collectible)
+                }
+                .buttonStyle(PlainButtonStyle())
             }
-        }.padding(.vertical)
+        }
+    }
+
+    var header: some View {
+        HStack {
+            TokenImage(width: imageDimention, height: imageDimention, url: section.imageURL, name: "ico-nft-placeholder")
+            Text(section.name).body().font(Font.gnoBody.weight(.semibold))
```

Comment:
instead of `.body().font(Font.gnoBody.weight(.semibold))` you should use `.headline()` - see the `Fonts.swfit` file